### PR TITLE
feat: add twitter image downloading script

### LINKED ISSUE #1003: Script to download Tweet Images
# Description

Create a Script to download all the Images associated with your tweets. All the Images should be download locally along with a CSV file accompanying them, which contains the Image ID and the tweet they were linked with.

## Language

- [X] Go
- [X] Javascript
- [X] Python

### PR DESCRIPTION
**Pull request by YashKandalkar**
# Description
Twitter images downloading script in python added.
Dependencies are listed in requirements.txt:
```
tweepy
decouple
requests
```

Fixes #1003

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots:
image

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Twitter_Images_Downloader/images_download.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    if downloaded == 0:
+        print("No images found")
+    else:
+        print(
+            f"Downloaded {downloaded} image(s) and written info in ./downloaded_images.csv")
+
+
+if __name__ == '__main__':
+    main()
```
Comment (RohiniRG) [inline Python/Twitter_Images_Downloader/images_download.py:83]: Leave a newline at the end of the file

File: Python/Twitter_Images_Downloader/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Twitter Images Downloader
+
+This script downloads images from given user's recent tweets and also creates a CSV file with `Tweet URL`, `Image ID`, `Image URL` columns.
+
+### Instructions
+
```
Comment (RohiniRG) [inline Python/Twitter_Images_Downloader/README.md:6]: Just add another instruction to install the requirements for running the script

**Review by RohiniRG — CHANGES_REQUESTED** [review]
Very well written code... Just make these tiny changes

File: Python/Twitter_Images_Downloader/README.md (hunk 2 above)
Comment (RohiniRG) [reply Python/Twitter_Images_Downloader/README.md:6]: Also, in the requirements.txt file, specify the versions of your packages.

Comment (YashKandalkar) [reply Python/Twitter_Images_Downloader/README.md:6]: Done

File: Python/Twitter_Images_Downloader/images_download.py (hunk 1 above)
Comment (YashKandalkar) [reply Python/Twitter_Images_Downloader/images_download.py:83]: done

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
@all-contributors bot please add @YashKandalkar for code

**Pull request merged**